modules/renter/files.go

```diff
@@ -88,10 +88,10 @@ func (f *file) numChunks() uint64 {
 }
 
 // available indicates whether the file is ready to be downloaded.
-func (f *file) available(isOffline func(types.FileContractID) bool) bool {
+func (f *file) available(contractStatus func(types.FileContractID) (bool, bool)) bool {
```

Comment:
the return values here should probably be named, confusing to have a `(bool, bool)) bool` construction

Reply:
done